ru/lessons/basics/pattern-matching.md

```diff
@@ -12,7 +12,7 @@ lang: ru
 
 ## Оператор сопоставления
 
-В языке Elixir оператор `=` называется оператором сопоставления. С его помощью можно как назначать, так и сравнивать значения:
+В языке Elixir оператор `=` на самом деле является оператором сопоставления по аналогии со знаком равенства в алгебре. Его использование превращает всё выражение в уравнение, и Elixir сопоставляет левую часть выражения с правой. В случае успешного сопоставления вернётся решение уравнения, иначе возникнет ошибка:
```

Comment:
Thanks, @brain-geek!
I don't clearly understand why do you suggest `само решенное уравнение` instead of `решение уравнения`. For example, `x = 1` returns and assigns the solution but not the equation itself.